components/cap-jurisdictions.js

```diff
@@ -0,0 +1,43 @@
+import { LitElement, html, css } from "../lib/lit.js";
+import { fetchJurisdictionsData } from "../lib/data.js";
+
+export default class CapJurisdictions extends LitElement {
+	static properties = {
+		jurisdictionsData: { attribute: false },
+	};
+
+	constructor() {
+		super();
+		this.jurisdictionsData = [];
+	}
+
+	connectedCallback() {
+		super.connectedCallback();
+		fetchJurisdictionsData((data) => (this.jurisdictionsData = data));
+	}
+
+	render() {
+		return html`
+			<article>
+				${Object.keys(this.jurisdictionsData)
+					.sort()
+					.map(
+						(jurisdiction) =>
+							html`<h2>${jurisdiction}</h2>
+								<ul>
+									${this.jurisdictionsData[jurisdiction].map(
+										(reporter) => html`<li>
+											<a href="/caselaw.html?reporter=${reporter.slug}"
+												>${reporter.short_name}</a
+											>: ${reporter.full_name}
+											(${reporter.start_year}-${reporter.end_year})
+										</li>`
+									)}
+								</ul>`
+					)}
+			</article>
```

Comment:
Super small change here. Each section can be independently wrapped in its own article, and then the heading inside that article identifies what the content is. For example: 

<article>
<h2>Arkansas</h2>
<ul>
<li>...reporter</li>
</ul>
</article>
<article>
<h2>Kentucky</h2>
</article>

![Group 1](https://github.com/harvard-lil/capstone-static/assets/4039311/74b1270a-6b05-49e0-a08a-728b1471a285)